pontoon/base/static/js/l20n_interface.js

```diff
@@ -0,0 +1,313 @@
+/* Public functions used across different files */
+var Pontoon = (function (my) {
+  return $.extend(true, my, {
+    l20n: {
+
+      /*
+       * Populate FTL translation area with existing data
+       */
+      renderEditor: function (translation) {
+        $('#ftl-area > .main-value').show().find('input').val('');
+        $('#ftl-area .traits ul:first').empty();
+
+        var entity = Pontoon.getEditorEntity(),
+            translation = translation || entity.translation[0],
+            isTranslated = translation.pk,
+            traits = isTranslated ? translation.extra.traits : entity.extra.traits,
+            key, namespace, value, builtIn;
+
+        // Main value
+        $('#ftl-area .main-value ul input').val($('#translation').val());
+
+        // Traits
+        if (traits) {
```

Comment:
`if (traits.length === 0) { return; }` would reduce the indent-level of the rest of the code.